high_card.rb

```diff
@@ -1,5 +1,45 @@
-# Use these two arrays to generate a deck of cards.
-ranks = ["A", 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K" ]
-suits = [ "hearts", "spades", "clubs", "diamonds" ]
+require "pry"
 
-players = []
+def build_deck
+@values = ["A", 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K" ]
```

Comment:
We aren't inside a Class,  therefore, we can't create an instance. Instance variable won't perform as expected.  I'm not sure how they act here.  Since you are only using them in this method, might as well make them local vars and signal that to the next developer.